src/tests/ftest/mpiio/llnl_mpi4py.py

```diff
@@ -28,8 +28,8 @@ def test_llnl(self):
         test_nb_readwrite, test_rdwr, test_readwrite
 
         :avocado: tags=all,pr,daily_regression
-        :avocado: tags=large
-        :avocado: tags=mpiio,smoke,mpich,llnl
+        :avocado: tags=vm
+        :avocado: tags=mpiio,smoke,mpich,llnl,test_llnl
```

Comment:
Minor. Feature tags vs test tags
```suggestion
        :avocado: tags=mpiio,smoke,mpich,llnl
        :avocado: tags=test_llnl
```